fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/TransactionImpl.java

```diff
@@ -30,40 +37,69 @@
  */
 public class TransactionImpl implements Transaction {
 
+    private final static Logger log = LoggerFactory.getLogger(TransactionImpl.class);
+
     final String id;
```

Comment:
Hmm, I think all the class variables should be declared as private unless they need to be protected